### PR TITLE
Add test coverage for GardensHelper

### PR DESCRIPTION
**Pull request by dv2**
Adding test coverage

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
Looks good to me. 

You might be interested in rspec's context blocks, which let you describe the test setup. You are already doing it with 

```
it "is less than 130 characters long"
```

but this lets you explain it better:

```
describe "garden description" do
  context "with a short description" do 
    before :each do
      # Whatever setup 
    end

     it 'renders everything'
     it 'has an x'
     it 'does y'
    end
  end
end
```

File: spec/helpers/gardens_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      result = helper.display_garden_plantings(plantings)
+
+      output = "<li>"
+      output += "10 " + link_to(crop.name, crop)
+      output += ", planted on #{plantings.first.planted_at}"
+      output += "</li>"
+      expect(result).to eq output
+    end
+
+    it "has 2 planting" do
```
Comment (cesy) [inline spec/helpers/gardens_helper_spec.rb]: I think this should be "has 2 plantings"

**Comment by cesy** [comment]
Odd that coveralls doesn't think coverage has increased. Oh well - this is good stuff. I've flagged up a couple of grammar nitpicks, and then we can merge. Thank you!

File: spec/helpers/gardens_helper_spec.rb (hunk 1 above)
Comment (dv2) [inline spec/helpers/gardens_helper_spec.rb]: @cesy I just updated this in 2 places, thanks for catching it

**Comment by dv2** [comment]
@CloCkWeRX I will use them going forward, unless you want me to update the specs now

**Comment by cesy** [comment]
Great, thank you!

**Pull request merged**